### ISSUE TITLE
duplicate() page - could/should set as active page?

### ISSUE THREAD

**Issue by ffd8** [basiljs/basil.js#357]
Was just messing with duplicating a page and got what seemed like a strange behavior... the duplicated page wasn't set as the active page... ie. on page 1.. duplicated.. added line.. was applied to page 1 instead of the duplicate, page 2.

At the moment, one has to set the duplicate as a var, then tell the `page()` to use that duplicate... I'd imagine if one is duplicating the page, they'd want it to become active too, no?

Currently necessary:
```
function draw() {
  var dup = duplicate(page());
  page(dup);
  line(0, 0, width, height);
}
```

Could/should be:
```
function draw() {
  duplicate(page());
  line(0, 0, width, height);
}
```

Justification... our `addPage()` both adds a page and sets it to active = nice in a loop.. seems like this should do the same. Thoughts?

**Comment by trych** [comment]
Yes, I agree, when a page is duplicated, we should jump to that page as well.

**Comment by trych** [comment]
Fixed in #359.
